tests/python/contrib/test_hexagon/test_launcher.py

```diff
@@ -40,6 +42,251 @@
 # triggering TIME_WAIT state on the server socket. This prevents another
 # server to bind to the same port until the wait time elapses.
 
+@requires_hexagon_toolchain
+def test_add_hvx(android_serial_number, tvm_tracker_host, tvm_tracker_port, adb_server_socket):
+    """
+    Starting with an elementwise-add computation, try various schedules / optimizations to
+    see the impact they have on performance.
+
+    The main motivation for this test is to explore the relationship between these
+    schedules / optimizations vs. how effectively the primfunc uses the Hexagon's
+    HVX units.
+    """
```

Comment:
Should this run benchmarks on the simulator as well as physical hardware?  If they should only run on physical hardware, can we add `if android_serial_number == 'simulator': pytest.skip()`?

Reply:
This is TBD I think.  Commit dbbbe4fb12840d3b1d2f791cde7d2fd993d800cc specializes timers on Hexagon to use `HAP_perf_get_time_us()`.   If the Hexagon simulator causes that function to return realistic values, I think the benchmark should still be viable in simulation.

I'll look into that and report back.

Reply:
Got it.  In that case, we should probably mark it with `@tvm.testing.slow` (introduced in #10057), because otherwise it would run in the pre-commit CI as part of `task_python_hexagon_simulator.sh`.

Reply:
> we should probably mark it with @tvm.testing.slow

This is moot if I'm moving this to a script that's not exercised by CI, correct?

Reply:
Update: I've confirmed that the Hexagon Simulator provides meaningful benchmark results via `time_evaluator`.  They don't perfectly line up with those we get on actual hardware, but they generally track the same way.

Reply:
Sounds good, so let's keep the simulator as an option.